src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java

```diff
@@ -0,0 +1,32 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class UTCLocalConverter {
+    public static String convertUTCtoLocal(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "yyyy-MM-dd'T'HH:mm:ss" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)